snuba/manual_jobs/toy_job.py

```diff
@@ -16,4 +18,10 @@ def _build_query(self) -> str:
             return "not dry run query"
 
     def execute(self) -> None:
-        click.echo("executing query `" + self._build_query() + "`")
+        click.echo(
```

Comment:
nit that doesn't block merge: we shouldn't use `click` anything in this namespace because that's specific to CLI usage, and in prod we expect this to be invoked by snuba admin (let's logger.info in a later PR)